### PR TITLE
feat: token switcher on neurons and proposals views

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

- Use the new "Token" switcher design on "Neurons" and "Proposals" views.
- Remove existing universe dropdown.

# PRs

- This PR -> #1677 -> `main`

# Changes

- remove select project dropdown components
- remove unused "select" property in `Summary`
- review `Summary` display / style
- extract common summary display with projects to `<SummaryUniverse />`
- add token switcher layout / split content to "Neurons" and "Proposals" views
- add universe / token title to neuron detail page

# Screenshots

Capture d’écran 2023-01-10 à 09 29 09
Capture d’écran 2023-01-10 à 09 29 15
Capture d’écran 2023-01-10 à 09 29 19
Capture d’écran 2023-01-10 à 09 29 24
Capture d’écran 2023-01-10 à 09 29 29
Capture d’écran 2023-01-10 à 09 29 32
Capture d’écran 2023-01-10 à 09 29 38
Capture d’écran 2023-01-10 à 09 29 42
Capture d’écran 2023-01-10 à 09 29 47

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
Just a comment.

Already approving, thanks!

**Review by mstrasinskis — APPROVED** [review]
LGTM, thank you!

File: frontend/src/lib/components/proposals/ProposalCard.svelte [hunk 1]
```diff
@@ -135,4 +135,11 @@
       --badge-color: var(--negative-emphasis-light);
     }
   }
+
+  .proposer {
+    @media (min-width: media.$breakpoint-extra-large) and (max-width: 1600px) {
+      max-width: 6vw;
+      @include text.truncate;
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply frontend/src/lib/components/proposals/ProposalCard.svelte:142]: That's a good idea 👍. Let me try

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply frontend/src/lib/components/proposals/ProposalCard.svelte:142]: Let's do that, it's more consistent. Done.

Capture d’écran 2023-01-10 à 11 28 51
Capture d’écran 2023-01-10 à 11 28 46

**Pull request merged**